### ISSUE TITLE
XRSubsystemHelpers issue with 2019.2.21f1

### ISSUE THREAD

**Issue by jbienzms** [microsoft/MixedRealityToolkit-Unity#8501]
## Describe the bug

In 2019.2.21f1 with the latest mrtk_develop branch I receive the following error:

> Assets\MRTK\Core\Utilities\XRSubsystemHelpers.cs(28,23): error CS0246: The type or namespace name 'XRInputSubsystem' could not be found (are you missing a using directive or an assembly reference?)

When opening in Visual Studio, it wants me to resolve the namespace as UnityEngine.XR

![image](https://user-images.githubusercontent.com/5230670/92543523-5608bb00-f211-11ea-8952-0fc0b95d2ccf.png)

However, the conditional compile statement in the code ([link](https://github.com/microsoft/MixedRealityToolkit-Unity/blob/77cfe8c5d4c7263d0f9ed2d6d80d54e288c9301d/Assets/MRTK/Core/Utilities/XRSubsystemHelpers.cs#L4)) says:

```
#if UNITY_2019_3_OR_NEWER
using System.Collections.Generic;
using UnityEngine;
using UnityEngine.XR;
#else
using UnityEngine.Experimental.XR;
#endif
```

I do not believe that UNITY_2019_3_OR_NEWER is the right selector for this.

Changing it to `#if UNITY_2019_2_OR_NEWER` resolves the `XRInputSubsystem` issue but causes problems for other types.

Changing the `#else` block above to include `using UnityEngine.XR;` does completely resolve the issue for 2.21f1, but probably opens a whole new set of issues for older versions.

## Your setup (please complete the following information)

- Unity Version: 2019.2.21f1
- MRTK Version: mrtk_development as of 9/8/2020

## Target platform (please complete the following information)

- HoloLens 2

**Comment by jbienzms** [comment]
Assigning @keveleigh as the creator of the file in a9ec39a75261d8fc500297b779b7919d6b36a081 and editor in a864bd38ff52d8f6a0dc2e6fefe14ecc465e550c. Please feel free to reassign.

**Comment by keveleigh** [comment]
My gut says that we might not make a change here, since we no longer support 2019.2 (since Unity has also dropped support for non-LTS builds), but I'll take a look.

**Comment by jbienzms** [comment]
Understood, and fair enough.

Just FYI and on a related note, our [Prerequisites](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/GettingStartedWithTheMRTK.html#prerequisites) page doesn't list any specific versions of Unity other than "2019". I was about to ask what version I _should_ be on, but then thought I would check the docs first.

**Comment by davidkline-ms** [comment]
> our Prerequisites page doesn't list any specific versions of Unity other than "2019".

We should update our prereqs to be unity 2018.4 or 2019.4 with a recommendation to be on 2019.4

**Comment by keveleigh** [comment]
Fixed via #8535
